website/content/en/docs/building-operators/golang/references/env-test-setup.md

```diff
@@ -10,29 +10,20 @@ This document describes how to configure the environment for the [controller tes
 
 ## Installing prerequisites
```

Comment:
I don't think we need the `Overview` and `Installing prerequisites` headers in this file. Right after the frontmatter, I would go right into this:

> By default, Go-based operators are scaffolded to make use of controller-runtime's [`envtest`][envtest] framework, which uses `kubectl`, `kube-apiserver`, and `etcd` to simulate the API portions of a real cluster. You can use [this script][script] to download ...